src/components/footer.tsx

```diff
@@ -1,13 +1,16 @@
-import Image from "next/image";
+type Props = {
+  className?: string;
+};
 
-const Footer = () => {
+const Footer = ({ className }: Props) => {
   return (
-    <div className="w-full flex items-center flex-col bg-lego">
-      <Image src="/images/footer.svg" alt="Footer" width={500} height={200} />
-      <div className="w-full bg-lego-muted flex flex-col items-center justify-center p-6 space-y-2">
-        <span className="text-white text-gray">Laget med 🍺 av Webkom</span>
-        <span className="text-white">© Abakus 2024</span>
-      </div>
+    <div className={`w-full ${className} text-black text-center mt-auto`}>
```

Comment:
Can we have some vertical padding on the footer please? @falbru

Reply:
NOOO